packages/augur-ui/src/modules/create-market/components/common.tsx

```diff
@@ -992,3 +996,85 @@ export const CategoricalTemplate = (props: CategoricalTemplateProps) => {
     </>
   );
 };
+
+export interface ResolutionRulesProps {
+  newMarket: NewMarket;
+  onChange: Function;
+}
+
+export const ResolutionRules = (props: ResolutionRulesProps) => {
+  const { onChange, newMarket } = props;
+  const { template } = newMarket;
+  const { resolutionRules } = template;
+  if (
```

Comment:
not following the logic here.  are you seeing if there is anything in that resolutionRules?

if (Object.keys(resolutionRules).length === 0)

that would do it.